src/mmpars.c

```diff
@@ -2590,148 +2595,149 @@ char parseProof(long statemNum)
+              bug(1736); // Make sure proof length didn't change
             }
             if (wrkProofString[k] != (g_WrkProof.proofString)[step]) {
-              bug(1737); /* Make sure future subproof is now the future local
-                            label (to be renumbered below) */
+              // Make sure future subproof is now the future local
+              // label (to be renumbered below).
+              bug(1737);
             }
 
-            /* We now have this wrkProofString[...] content:
-                [0]...[step-1]                   same as original proof
-                [step+1]...[step+m-1]            moved subproof
-                [step+m]...[k-1]                 shifted orig proof
-                [k]...[k]                        subproof replaced by loc label
-                [k+1]...[g_WrkProof.numSteps-1]    same as orig proof */
+            // We now have this wrkProofString[...] content:
+            //    [0]...[step-1]                   same as original proof
+            //    [step+1]...[step+m-1]            moved subproof
+            //    [step+m]...[k-1]                 shifted orig proof
+            //    [k]...[k]                        subproof replaced by loc label
+            //    [k+1]...[g_WrkProof.numSteps-1]    same as orig proof
 
-            /* Correct all local labels */
+            // Correct all local labels
             for (i = 0; i < g_WrkProof.numSteps; i++) {
-              j = (wrkProofString)[i]; /* Will be <= -1000 if local label */
-              if (j > -1000) continue; /* Not loc label ref */
-              j = -1000 - j; /* Restore step number subproof ends at */
-              /* Note: the conditions before the "&&" below are redundant
-                 but provide better sanity checking */
-              if (j >= 0 && j < step) { /* Before moved subproof */
-                /*j = j;*/ /* Same as orig proof */
-              } else if (j == step) { /* The original local label ref */
-                bug(1738); /* A local label shouldn't ref a local label */
-              } else if (j > step && j <= k - m) {
-                                   /* Steps shifted up by subproof insertion */
-                j = j + m - 1; /* Offset by size of subproof moved down -1 */
+              j = (wrkProofString)[i]; // Will be <= -1000 if local label
+              if (j > -1000) continue; // Not loc label ref
+              j = -1000 - j; // Restore step number subproof ends at
+              // Note: the conditions before the "&&" below are redundant
+              // but provide better sanity checking.
+              if (j >= 0 && j < step) { // Before moved subproof
+                // j = j; // Same as orig proof
+              } else if (j == step) { // The original local label ref
+                bug(1738); // A local label shouldn't ref a local label
+              // Steps shifted up by subproof insertion
+              } else if (j > step && j <= k - m) {          
```

Comment:
```suggestion
              } else if (j > step && j <= k - m) { // Steps shifted up by subproof insertion
```
if this goes over the line length put it in the block

Reply:
Alright, I'll probably have to manually change some of those in #153 then.